package.json

```diff
@@ -64,7 +67,7 @@
     "cozy-editor-core": "134.0.3",
     "cozy-flags": "2.7.2",
     "cozy-realtime": "3.13.0",
-    "cozy-scripts": "5.10.0",
+    "cozy-scripts": "npm:anc-scripts@^6.0.0",
```

Comment:
Nein. We don't use personal fork. If you want to use a new version of cozy-scripts or else, you can publish a new tag for our cozy-scripts lib and use it here. You can create a `cozy-script@eslint7" or something

Reply:
Yes I was expecting that, will do

Reply:
updated